BrainPortal/app/controllers/groups_controller.rb

```diff
@@ -132,7 +132,7 @@ def create  #:nodoc:
   # PUT /groups/1.xml
   # PUT /groups/1.json
   def update #:nodoc:
-    @group = current_user.available_groups.where( :type => "WorkGroup" ).find(params[:id])
+    @group = current_user.available_groups.find(params[:id])
```

Comment:
No, this will allow the admin from modifying groups other than WorkGroups (like SiteGroups). Before, the update action could not find groups other than WorkGroups, so the admin could not update them.